deploy/runtools/firesim_topology_with_passes.py

```diff
@@ -9,16 +9,16 @@
 from runtools.switch_model_config import *
 from runtools.firesim_topology_core import *
 from runtools.utils import MacAddress
-from fabric.api import *
-from colorama import Fore, Style
+from fabric.api import * # type: ignore
+from colorama import Fore, Style # type: ignore
 import types
 from functools import reduce
 
 from util.streamlogger import StreamLogger
 
 rootLogger = logging.getLogger()
 
-@parallel
+@parallel # type: ignore
```

Comment:
How does mypy complain if you don't put this here?  Does this mean that for all uses of the `@parallel` decorator we have to add this type pragma?

Reply:
I got the following error when I removed the `#type: ignore`: `deploy/runtools/firesim_topology_with_passes.py:21: error: Name "parallel" is not defined`. In other files, I had to add the ignore to the `import fabric.api` lines as well. I assume that once we upgrade to newer Fabric this will go away?

Reply:
I got curious about this and started building some stubs for Fabric.  We don't really need to include them in this PR.  I'd rather get them upstream somewhere before we start using them.  Just wanted to mention that I have them.